webapp/channels/src/packages/mattermost-redux/src/reducers/entities/ai.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {combineReducers} from 'redux';
+
+import type {MMReduxAction} from 'mattermost-redux/action_types';
+
+import {AITypes} from '../../action_types';
+import type {AIAgent} from '../../actions/ai';
+
+export interface AIState {
+    agents: AIAgent[];
+}
+
+function agents(state: AIAgent[] = [], action: MMReduxAction): AIAgent[] {
```

Comment:
Should something happen here if the plugin is turned off? Based on the other conversation that stated the presence of these agents is the on/off switch for AI features, we should probably blow these out if the plugin is turned off.